app/views/messages/_new_message.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for @newMessage, url: messages_path do |f| %>
+  <div class="field">
+    <!-- Sender should be calculated automatically -->
+    <input id="sender" name="message[sender]" type="hidden" value="{{myCountry}}" />
```

Comment:
Part of the reason that we want to know what country we're in. It makes calculating fields for replies really easy.